packages/theme-language-server-common/src/test/CompletionItemsAssertion.spec.ts

```diff
@@ -19,12 +19,12 @@ describe('Module: CompletionItemsAssertion', () => {
     });
   });
 
-  it('should assert a list of labels', () => {
-    expect(provider).to.complete('{% rend', ['render']);
+  it('should assert a list of labels', async () => {
+    await expect(provider).to.complete('{% rend', ['render']);
   });
 
-  it('should assert a list of completion items', () => {
-    expect(provider).to.complete('{% rend', [
+  it('should assert a list of completion items', async () => {
+    await expect(provider).to.complete('{% rend', [
```

Comment:
without the `await`, the test was always passing even when i changed the expected result